### ISSUE TITLE
Could you please share pre-trained model?

### ISSUE THREAD

**Issue by ishansan38** [codetendolkar/tacotron-2-explained#1]
Hi @codetendolkar Hope you are doing good.

Thanks for the implementation of Taco2, I was wondering if it was possible to share a pre-trained model for me to check the audio samples before training it myself. I don't have access to high computation right now and would be awesome if you could share it.

Best,
Ishan,

**Comment by gauravtendolkar** [comment]
Hi Ishan, I am currently tweaking and training it to get benchmark level results. I hope I can get them by end of this month. 

Right now, the model on this repo has no batch norm layers. The paper is not very clear about some parameters. Also, I average all the channels of the postnet output (residuals) and add it to input of postnet (mels) for secondary loss. I have equal weights for both losses

I would also modify the structure to use the Estimator API so that its easier for everyone else

I will attach a pre-trained model with dataset asap. Till then I hope the comments are instructive enough to try it out on your own :)

**Comment by sujeendran** [comment]
Hi, 
Do we have any update on a pre-trained model? I would really like to test and see if I can run the model on OpenVINO framework and is currently limited on computation power for training.

**Comment by TheButlah** [comment]
Any updates on the pre-trained model? I'm considering using this as a part of a larger system and having the pre-trained model would go a long way towards verifying the accuracy of this implementation.
